packages/app/models/order/types.ts

```diff
@@ -1,4 +1,4 @@
-import { Order as StacklyOrder } from "@stackly/sdk/dist/vaults/subgraph";
+import { Order as StacklyOrder } from "@stackly/sdk";
 
 export interface Order extends StacklyOrder {}
```

Comment:
Still can't understand why we are doing this. We are importing from `Order` from sdk, renaming it to `StacklyOrder`, extending `StacklyOrder` to a new interface called `Order`. Which takes us back to the initial `Order` importing that's equal to the new `Order`.

Reply:
I think we can use Order from `sdk` now, will update it.